prow/flagutil/github_test.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package flagutil
+
+import (
+	"testing"
+)
+
+func TestGithubOptions(t *testing.T) {
+
+	testCases := []struct {
+		name        string
+		endpoints   Strings
+		gitEndpoint string
+		tokenPath   string
+		expectedErr bool
+	}{
+		{
+			name:        "No token provided, we expect no errors",
```

Comment:
Why shouldn't this error?

Reply:
I assume because it was the initial behavior. Which seems to be correct, because anonymous git access is a thing.

Reply:
Let's open an issue to fix this, it's not viable to run out components anonymously.